lib/Doctrine/Common/Annotations/AnnotationRegistry.php

```diff
@@ -142,10 +150,13 @@ static public function loadAnnotationClass($class)
         }
 
         foreach (self::$loaders AS $loader) {
-            if (call_user_func($loader, $class) === true) {
+            if ($loader($class) === true) {
                 return true;
             }
         }
+
+        self::$failedToAutoload[$class] = null;
```

Comment:
The lookup can happen a lot of times (thousands according to the original report), so I'd rather keep it indexed